doc/plutus-core-spec/versions.tex

```diff
@@ -0,0 +1,72 @@
+\section{Version numbers and the Cardano blockchain}
+\label{appendix:version-numbers}
+After a Plutus Core script has been deployed to the blockchain it may be
+re-evaluated at any point in the future when the history of the chain is
+verified.  Plutus Core may evolve as time progresses, and because of this we
+have to be quite careful with versioning: a detailed discussion of this issue
+can be found in~\cite{CIP-35}.  There are two types of version which are
+relevant in this document:
+
+\begin{itemize}
+\item The language itself may evolve: for example some fundamental new construct
+  may be added.  A \textit{Plutus Core language version} describes a version of
+  the basic language with a particular set of features. A language version
+  consists of three non-negative integers separated by decimal points, for
+  example \texttt{1.4.2}.  Language versions are ordered lexicographically.  As
+  we have seen in Section~\ref{sec:grammar-notes}, the Plutus Core
+  language version used by a script is included in the \texttt{program} header.
+  
+\item The built-in types and functions deployed on the chain along with a
+  particular version of the Plutus Core language can change with time.  Plutus
+  Core scripts on the Cardano chain are equipped with a tag called a
+  \textit{ledger language version} which (among other things) determines which
+  set of types and functions are to be used during evaluation of the script.
+  Ledger language versions have names names like ``Plutus V1'', ``Plutus V2'',
```

Comment:
Here and throughout: I strongly suggest using "PlutusV1" over "Plutus V1". The latter suggests that there are two components, and one of them is a "version". This is not true: "PlutusV1" is just an uninterpreted string, that we happen to have given a descriptive name. Or to put it another way, the ledger language _is_ the string "PlutusV1" not "Plutus V1".